kraken/node_actions/abstract_node_scenarios.py

```diff
@@ -7,17 +7,22 @@
 class abstract_node_scenarios:
 
     # Node scenario to start the node
-    def node_start_scenario(self, instance_kill_count, node, timeout):
+    def node_start_scenario(
+        self,
+        instance_kill_count,
+        node,
+        timeout,
+    ):
         pass
 
     # Node scenario to stop the node
-    def node_stop_scenario(self, instance_kill_count, node, timeout):
+    def node_stop_scenario(self, instance_kill_count, node, timeout, force_action=False):
         pass
 
     # Node scenario to stop and then start the node
-    def node_stop_start_scenario(self, instance_kill_count, node, timeout):
+    def node_stop_start_scenario(self, instance_kill_count, node, timeout, force_action=False):
         logging.info("Starting node_stop_start_scenario injection")
-        self.node_stop_scenario(instance_kill_count, node, timeout)
+        self.node_stop_scenario(instance_kill_count, node, timeout, force_action)
```

Comment:
@chaitanyaenr @mffiedler any ideas on how to rework this so I don't have to go through and add "force_action" as a parameter in all the stop scenario function definitons in each cloud? Want to add in the option for alibaba specifically and ignore for all other clouds

Works for alibaba but get the below error on AWS. 
`TypeError: node_stop_scenario() takes 4 positional arguments but 5 were given`

Reply:
I would go with not defining the force_action as parameter to keep things generic for all cloud providers and instead set force_action=true by default for Alibaba node actions as that might be more representative of chaos. Thoughts?

Reply:
Set default force value to True